packages/components/Footer.tsx

```diff
@@ -1,25 +1,32 @@
 import { Stack, Text, useColorModeValue, Link } from '@chakra-ui/react';
+import QRCode from 'react-qr-code';
 
 import { TranslationNS, useI18n } from '@packages/features/i18n-context';
+import { QrCodeTextGenerator, PixKey } from '@packages/config';
```

Comment:
Uma convenção em JS é nomear as constantes com upper case e usando um `_` entre as palavras, tipo 
```suggestion
import { PIX_QR_CODE, PIX_KEY } from '@packages/config';
```